iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -124,4 +124,16 @@
     <PackageReference Condition=" '$(Configuration)' == 'Debug' " Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
+    <PackageReference Include="Microsoft.Win32.Registry">
+      <Version>4.5.0</Version>
+    </PackageReference>
```

Comment:
should this info be same for linux ?

Reply:
It seems like it's fine.  It compiled successfully and passed the unit tests on Linux (once I added the check for Windows OS in the code).  I'm not very familiar with writing cross-platform code, so don't hesitate to let me know if there's something that can be improved.